docs/source/guide/glossary.md

```diff
@@ -0,0 +1,59 @@
+tuning a physical device so that it better approximates some ideal property or operation.)
+
+Expectation Value
+: The expectation value of an observable $A$ on state $\rho$ is the average
+readout value when $A$ is measured on $\rho$. Mathematically, this is $\text{Tr}[A\rho]$ and usually denoted $\langle A \rangle$ (when this notation is used, the state $\rho$ that $A$ is being measured on should be clear from context). Expectation values are important 
+for near-term quantum computing because in variational quantum algorithms,
+the only role of the quantum processor is to repeatedly compute expectation values, which a classical processor then uses to perform some overall useful computational task. In Mitiq, [Executors](executors.md) are used to calculate error-mitigated expectation values.
+
+Gate Fidelity
+: A number between 0 and 1 measuring how 
+closely a particular device's (noisy) physical implementation of a gate approximates the ideal gate's action on quantum states. Mitiq implements a noise-scaling method for ZNE in which each gate of the input circuit is sampled for [unitary folding](./zne-3-options.md#unitary-folding) with probability proportional to its infidelity (1 - fidelity), described [here](./zne-3-options.md#folding-gates-by-fidelity) and [here](https://mitiq.readthedocs.io/en/stable/apidoc.html#mitiq.zne.scaling.folding.fold_gates_at_random) in the documentation.
+
+Hamiltonian
+: A Hermitian operator whose eigenvalues and eigenvectors represent, respectively, a quantum system's possible energy levels and corresponding energy states. Most variational quantum algorithms work by encoding the objective of an optimization problem (e.g. finding the maximum cut in a graph) as the task of minimizing the expectation value of a problem-specific Hamiltonian, which physically corresponds to finding the ground-state energy of that Hamiltonian. For an example of how error mitigation helps such algorithms, see [Solving MaxCut with Mitiq-improved QAOA](../examples/maxcut-demo.md).
+
+Sampling Overhead
+: The basic resource-cost measure used to evaluate QEM methods---how many more circuit executions ("runs," "shots") does a method need to achieve the same level of statistical precision in estimating an expectation value, compared to 
+the naive (i.e. unmitigated) method of running the same noisy input circuit $N$ times and returning the
+sample mean of the measurement outcomes. Also called sampling cost, it is usually reported as a multiplicative factor $C$, defined as the ratio of the QEM estimator's variance to the sample-mean estimator's variance, and meaning that
+the method needs $C \cdot N$ circuit shots to obtain the same precision as the sample-mean estimator would with only $N$ shots. 
+
+
+## QEM Methods
+[Clifford Data Regression (CDR)](cdr.md)
+: An error mitigation model is
+trained with quantum circuits that resemble the circuit of interest, but which are easier to classically simulate. 
```

Comment:
What's up with the `:` that begins the description of each entry in the glossary? Is it needed? If not, let's remove it.

Reply:
I think it's required to get this "definition list" format for the entries like in pennylane's glossary. Here is where I got the syntax from: https://myst.tools/docs/mystjs/typography#definition-lists

Reply:
Ah okay I missed that! Thanks for the link. Great syntax to know going forward.